src/platform.ts

```diff
@@ -69,14 +71,17 @@ export class LinuxDistribution {
                 else if (key === 'VERSION_ID') {
                     version = value;
                 }
+                else if (key === 'ID_LIKE') {
+                    idLike = value.split(" ");
+                }
 
                 if (name !== unknown && version !== unknown) {
```

Comment:
We ignore ID_LIKE if ID is something we understand. If we don't understand ID, we will iterate through the ID_LIKE values to see if we understand any of them.